### PR TITLE
Logout category and testlogin

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey please review my small PR for logout and theme

[Click here](https://github.com/bhushannaikaroskar/ecom-react/pull/5/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true#diff-0a3d322d21b6a0300f05ce8bb51e5ec518f0e0bbd67949dc91d249a4a36805d3)

Ignore these files:
 /backend folder
ProductFilter.jsx inisde ProductPage folder.

### CODE REVIEW

File: src/components/ProductPage/ProductFilter.jsx [hunk 1]
```diff
@@ -2,13 +2,17 @@ import { useProductContext } from "../../context/productContext/ProductProvider"
 
 export default function ProductFilters() {
     const { state, dispatch } = useProductContext();
+    const {maxPrice,sort,rating,category} = state
```
Comment (varun13169) [inline src/components/ProductPage/ProductFilter.jsx:5]: nit: spacing between variables.

File: src/components/WishListCard.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react";
 import { useCartContext, useWishListContext } from "../context";
 
 export default function WishListCard({ productData }) {
-    const { title, description, price, imageSrc } = productData;
+    const { title,rating, description, price, imageSrc } = productData;
```
Comment (varun13169) [inline src/components/WishListCard.jsx:5]: nit: spacing between rating

**Review by varun13169 — APPROVED** [review]
Functionality is neatly done, code beautification can be done better

**Review by humzahasan — APPROVED** [review]
Code changes looks good! 

A suggestion to have your components starting with Upper case. You missed that in few! also for further refactoring separate components and pages!

Good to merge!

**Review by zoyatanweer — APPROVED** [review]
- Clean code
- Naming of files and folder should be consistent , someplace it is lowercase and someplace it is in uppercase
- good to merge

**Pull request merged**